app/views/automated_tests/student_interface.html.erb

```diff
@@ -4,15 +4,15 @@
 <%= javascript_include_tag 'result', nonce: true %>
 <%= javascript_include_tag 'Results/main', nonce: true %>
 
-<% if @assignment.enable_student_tests && !@grouping.nil? %>
+<% if @assignment.enable_student_tests && @grouping.present? %>
```

Comment:
Revert this change (and the one below) since it's in a separate pull request. Note that you can modify the token end date in the automated tests settings tab for testing purposes.